packages/css/src/components/gap/gap.scss

```diff
@@ -14,7 +14,7 @@
   @if $size != "no" {
     .ams-gap--#{$size} {
       /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
-      grid-gap: var(--ams-gap-#{$size}) !important;
+      grid-gap: var(--ams-space-#{$size}) !important;
```

Comment:
If `grid-gap` is not a css rule why define it here? If you're going to use it as a util for gap then use `gap:`, it will also work with flexbox.

Reply:
Also, `!important` should no be needed if you use this as a class right?